README.md

```diff
@@ -7,7 +7,7 @@
 Visit [prometheus.io](https://prometheus.io) for the full documentation,
 examples and guides.
 
-Prometheus is a systems and service monitoring system. It collects metrics
+Prometheus, a [Cloud Native Computing Foundation](https://cncf.io/) project, is a systems and service monitoring system. It collects metrics
```

Comment:
I guess the point of branding is to place it quite prominent. Given the support we get from CNCF, I'm personally fine having it here.